astropy_timeseries/io/kepler.py

```diff
@@ -1,37 +1,98 @@
-from astropy.io import registry
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import warnings
+
+import numpy as np
+
+from astropy.io import registry, fits
 from astropy.table import Table
-from astropy.time import Time
+from astropy.time import Time, TimeDelta
 
-from ..sampled import TimeSeries
+from astropy_timeseries.sampled import TimeSeries
 
-__all__ = ['kepler_fits_reader']
+__all__ = ["kepler_read"]
 
 
-def kepler_fits_reader(filename):
+def kepler_read(filename):
```

Comment:
Can you change this back to ``kepler_fits_reader``? (I think there is also a non-FITS format for Kepler, which we don't need to worry about for now, but just for future-proofness)